examples/programming-lang-detection/train.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import data_helper
+from keras.models import Sequential, Model
+from keras.layers import Activation, Dense, Dropout, Flatten, Input, Conv1D, MaxPooling1D
+from keras.layers.merge import Concatenate
+import defs
+
+
+np.random.seed(2)
+
+# data parameters
+n_max_files = 2000 # max number of files per class to load
+number_of_classes = len(defs.langs)
+
+# Model Hyperparameters
+sequence_length = defs.file_characters_truncation_limit
+filter_sizes = (3, 9, 19)
+pooling_sizes = (3, 9, 19)
+num_filters = 128
+dropout_prob = (0.25, 0.5)
+hidden_dims = 128
+number_of_quantised_characters = len(defs.supported_chars)
+
+
+# Training parameters
+batch_size = 100
+num_epochs = 20
+val_split = 0.1
+
+
+# Data Preparatopn
+# ==================================================
+#
+# Load data
+print("Loading data...")
+x, y, z = data_helper.get_input_and_labels(file_vector_size=sequence_length, max_files=n_max_files, breakup=True)
+
+# Shuffle data
+shuffle_indices = np.random.permutation(np.arange(len(y)))
+x_shuffled = x[shuffle_indices]
+y_shuffled = y[shuffle_indices]
+
+# Setting up the model
+graph_in = Input(shape=(sequence_length, number_of_quantised_characters))
```

Comment:
this part can be simplified, it's using functional model API, and plug into sequential model API.
You can just write:
```
model = Sequential()
model.add(some layers)
for i in range(0, len(filter_siezes)):
    model.add(Conv1D( ... ))
...
```

refer to https://keras.io/getting-started/functional-api-guide/
and https://keras.io/getting-started/sequential-model-guide/